man/sphinx.md

```diff
@@ -25,382 +26,303 @@ echo -e "oldpassword\nnewpassword" | `sphinx` change \<user> \<site> [\<u\>\<l\>
 
-If all goes well, there is no output expected. If anything goes wrong,
-there is going to be an error message and a non-zero exit-code.
+If the operation is successful, there will be no output. If an error occurs, an error message will be displayed and the command will exit with a non-zero status code.
 
 ## UNDOING A PASSWORD COMMIT
 
-If you somehow messed up and have to go back to use the old password, you can
-undo committing your password using:
+If you need to revert to the old password after committing a new one, run:
 
 ```
 echo -n 'my master password' | sphinx undo username example.com
 ```
 
-Depending on your `rwd_keys` configuration setting, you might have to
-provide your master password on standard input to this operation.
+Depending on your `rwd_keys` configuration, you may need to provide your master password on standard input for this operation.
 
-If all goes well, there is no output expected, otherwise there will be
-an error message and non-zero exit-code.
+If successful, there will be no output. Otherwise, an error message and a non-zero exit code will indicate a problem.
 
 ## DELETING PASSWORDS
 
-In case you want to delete a password, you can do using the following
-command:
+To delete a password, use the following command:
 
 ```
 echo -n "my master password" | sphinx delete username example.com
 ```
 
-You provide the `delete` operation as the first parameter to the
-client, your `username` as the 2nd and the `site` as the 3rd
-parameter. This command does not provide any output on the console in
-case everything goes well, otherwise an error message and an non-zero
-exit code will signal a problem.
+Here:
+
+- `delete` specifies the operation.
+- `username` is your account name.
+- `example.com` is the target site.
 
-Depending on your `rwd_keys` configuration setting, you might have to
-provide your master password on standard input to this operation.
+If the operation succeeds, there will be no output. If it fails, you will see an error message and a non-zero exit code. Depending on your `rwd_keys` configuration, you may need to provide your master password via standard input.
 
 ## QR CODE CONFIG
 
-In case you want to use phone with the same sphinx server, you need to
-export your config to the phone via a QR code.
+If you want to use your phone with the same SPHINX server, you can export your configuration to the phone using a QR code.
 
 ```
 sphinx qr
 ```
 
-Will display a QR code containing only public information - like the
-server host and port, and whether you use rwd_keys. This is mostly
-useful if you want to share your setup with a friend or family.
+This displays a QR code containing only public information, such as the server host, port, and whether `rwd_keys` is enabled. This is useful for sharing your setup with trusted friends or family.
```

Comment:
actually the `[servers]` section, and also other settings, like userlist, validate_password, delete_upgraded and threshold.